src/main/cpp/t04_count.cpp

```diff
@@ -23,10 +23,23 @@
 
 #include "t04_count.h"
 #include <iostream>
-
+#include <string>
 
 using namespace std;
 
 int t04_count() {
+	string s;
+	getline(cin, s);
+	int a = s.length();
+	int count = 1;
+	int max = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'max' is assigned a value that is never used.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4951806)